presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcPageSource.java

```diff
@@ -273,6 +287,12 @@ private static Block buildNullBlock(Type type)
         return blockBuilder.build();
     }
 
+    private static Block buildSingleValueBlock(Type type, UUID shardUuid)
```

Comment:
Make this take a `Slice` and do the conversion in the caller (so this method is more generic)